src/GoTrueClient.ts

```diff
@@ -186,7 +187,7 @@ export default class GoTrueClient {
    * @param scopes A space-separated list of scopes granted to the OAuth application.
    */
   async signIn(
-    { email, phone, password, refreshToken, provider }: UserCredentials,
+    { email, phone, password, refreshToken, provider, openIDConnect }: UserCredentials,
```

Comment:
```suggestion
    { email, phone, password, refreshToken, provider, oidc }: UserCredentials,
```

I am seeing more use of `oidc`in packages like https://github.com/panva/node-oidc-provider/blob/9da61e9c9dc6152cd1140d42ea06abe1d812c286/lib/actions/authorization/device_user_flow_response.js#L15

Perhaps use the abbreviation?